### ISSUE TITLE
Atom IDE UI logo

### ISSUE THREAD

**Issue by simurai** [facebooknuclide/atom-ide-ui#1]
Everything needs a logo, let's see if we find one for Atom IDE UI. Will be use in this repo's README and maybe other places as well.

> Note that it's largely a subset of Nuclide and a logo that paid homage in some way might be cool.

Below some explorations.

**Comment by simurai** [comment]
## Concept A

![image](https://user-images.githubusercontent.com/378023/28957815-ae21f82e-792e-11e7-9b53-0facc171618b.png)

Ok, this is pretty close to Nuclide's style. It should resemble `UI` inside the :dizzy: circle. Not sure if it's **too** close and waters down Nuclide's brand?

**Comment by matthewwithanm** [comment]
@simurai you're the coolest 😊

**Comment by captbaritone** [comment]
Nice!

**Comment by iolsen** [comment]
I really like it! But the whole homage to Nuclide thing was my suggestion so I'm open to alternative approaches.

**Comment by karincurkowicz** [comment]
I'm wondering how this will be perceived by the Atom user base? Are they familiar with the Nuclide logo & product, or will this style change (from the green of Atom) be confusing for them? 

What do you all think the Atom user's perception be of the Atom IDE? How much of the Nuclide identity should be involved with that perception? 

- I'm still learning about the relationships, products, and users, so perhaps you can enlighten me.

**Comment by damieng** [comment]
I think we probably just want a logo/branding around Atom IDE rather than "Atom IDE UI" - the latter is half the story - the other half being the IDE-xxx packages and the behind-the-scenes atom-languageclient bridge.

**Comment by iolsen** [comment]
> I'm wondering how this will be perceived by the Atom user base? Are they familiar with the Nuclide logo & product, or will this style change (from the green of Atom) be confusing for them?

The majority of Atom's users are not familiar with Nuclide. I'm not sure _confusion_ is the primary concern, but I think you're suggesting something else that's true: that the branding should communicate the GitHub team is, in part, behind this. But giving the Nuclide team credit would also be great. Maybe a logo that's a fusion of the Atom and Nuclide logos? I'm not sure what that might look like. 😆 

> I think we probably just want a logo/branding around Atom IDE rather than "Atom IDE UI" - the latter is half the story - the other half being the IDE-xxx packages and the behind-the-scenes atom-languageclient bridge.

I agree, but where would that live? My thinking was that Atom IDE UI is the user-visible gateway. But  I'd forgotten to account for our intent to eventually bundle `atom-ide-ui`. Any branding associated with the package itself and appearing on atom.io is virtually irrelevant when that happens. So maybe we should be talking about a mini-site much like the GitHub package's. How do you feel about that, @simurai?

**Comment by simurai** [comment]
> @damieng I think we probably just want a logo/branding around Atom IDE rather than "Atom IDE UI"

I see. Yeah, having some sort of umbrella brand for all the different pieces sounds like a good idea.

> @iolsen So maybe we should be talking about a mini-site much like the GitHub package's. How do you feel about that, @simurai?

Sounds good too. The content could be similar to the blog post?

An alternative could also be a section on atom.io's home page. But maybe only after it got bundled in Atom.

**Comment by iolsen** [comment]
> The content could be similar to the blog post?

That's a good starting point, agreed.

> An alternative could also be a section on atom.io's home page.

I'd be OK with that, too. I just really like the way the GitHub package's mini-site turned out, and it's a nice, clean separation. I think it's a good way to hype significant new feature areas.

**Comment by karincurkowicz** [comment]
I agree @simurai & @iolsen, the umbrella brand concept makes sense. 

I recommend, from a visual standpoint, staying within the Atom "Green" color palette (for branding consistency, and using more subtle elements to reference Nuclide's contribution. 

I also would recommend avoiding the vertical UI combination as it has connotations of rude gestures and phallic symbols. ;-)

Obviously, this wouldn't be appropriate for a more umbrella Atom brand, but hopefully, it moves the conversation forward. 

![nuclideide](https://user-images.githubusercontent.com/22400181/29051602-bc126858-7b98-11e7-8262-5f8b2049b2b8.png)

**Comment by iolsen** [comment]
> connotations of rude gestures and phallic symbols

Wow now that you mention this, I see it and can't unsee it. Good catch.

**Comment by simurai** [comment]
> connotations of rude gestures and phallic symbols

Ouch.. 😆 🙈  Thanks for preventing further disaster. ❤️ 

I wonder if we should move away from using `UI` altogether? And maybe keep it as simple as saying `IDE` together with the Atom logo. Similar how Apple and other companies use sub-brands.

![image](https://user-images.githubusercontent.com/378023/29058935-16d2fbbc-7c4e-11e7-9e2e-afc372b4c9e8.png)

For the mini site, we could also spell it out as `Introducing Atom IDE`. And have the individual parts of the Nuclide logo spread around like 🎉  confetti:

![image](https://user-images.githubusercontent.com/378023/29059073-b8f03392-7c4e-11e7-9106-82e916deb8e3.png)

**Comment by wbinnssmith** [comment]
this looks sooo good! excited!! 🎊
